devito/core/cpu.py

```diff
@@ -203,8 +201,7 @@ def _specialize_clusters(cls, clusters, **kwargs):
         template = lambda: "r%d" % counter()
 
         # Toposort+Fusion (the former to expose more fusion opportunities)
-        clusters = Toposort().process(clusters)
-        clusters = fuse(clusters)
+        clusters = fuse(clusters, toposort=True)
```

Comment:
why not directly `topofuse`?

Reply:
because `fuse` is called later when there's no need for topo-sorting